assets/xml/scenes/overworld/spot04.xml

```diff
@@ -2,8 +2,26 @@
     <File Name="spot04_scene" Segment="2">
         <Path Name="spot04_scenePathList_00030C" Offset="0x030C" NumPaths="3"/>
         <Path Name="spot04_scenePathList_00D728" Offset="0xD728" NumPaths="2"/>
-        <Cutscene Name="gKokiriForestDekuSproutCs" Offset="0xC9D0"/>
-        <Cutscene Name="gSpot04Cs_10E20" Offset="0x10E20"/>
+        <Cutscene Name="gKokiriForestSariaGreetingCs" Offset="0x10E20"/>
+        <Cutscene Name="gKokiriForestIntroNaviFlyingCs" Offset="0xA6D0"/>
+        <Cutscene Name="gKokiriForestUnusedCs" Offset="0xB280"/>
+
+        <Cutscene Name="gKokiriForestCreditsDekuSproutCs" Offset="0xB9B0"/>
+        <Cutscene Name="gKokiriForestCreditsCameraPanCs" Offset="0xBB20"/>
+        <Cutscene Name="gKokiriForestCreditsCameraPanUnusedCs" Offset="0xB800"/>
+
+        <Cutscene Name="gKokiriForestDekuSproutPart1Cs" Offset="0xBCB0"/>
+        <!-- gCutsceneMapDekuSproutPart2Cs -->
+        <Cutscene Name="gKokiriForestDekuSproutPart3Cs" Offset="0xC9D0"/>
+
+        <Cutscene Name="gKokiriForestKokiriEmeraldPart1Cs" Offset="0x8DA0"/>
+        <!-- gCutsceneMapKokiriEmeraldPart2Cs -->
+        <!-- gCutsceneMapKokiriEmeraldPart3Cs -->
+        <!-- gGerudoValleyKokiriEmeraldPart4Cs -->
+        <!-- gGerudoValleyKokiriEmeraldPart5Cs -->
+        <Cutscene Name="gKokiriForestKokiriEmeraldPart7Cs" Offset="0x8950"/>
+        <!-- gCutsceneMapKokiriEmeraldPart8Cs -->
```

Comment:
imo the comments arent needed. if someone sees this and goes "where are the other parts" they can do a global search and will see the rest of them in the cutscene map file